posthog/api/capture.py

```diff
@@ -534,23 +541,122 @@ def get_event(request):
                             sent_at,
                             event_uuid,
                             token,
-                            extra_headers=[("lib_version", lib_version)],
+                            lib_version,
                         )
                     )
 
                 start_time = time.monotonic()
                 for future in futures:
-                    future.get(timeout=settings.KAFKA_PRODUCE_ACK_TIMEOUT_SECONDS - (time.monotonic() - start_time))
-
+                    if future is not None:
+                        future.get(timeout=settings.KAFKA_PRODUCE_ACK_TIMEOUT_SECONDS - (time.monotonic() - start_time))
+    except ValueError as e:
+        with sentry_sdk.push_scope() as scope:
+            scope.set_tag("capture-pathway", "replay")
+            scope.set_tag("ph-team-token", token)
```

Comment:
I am almost sure this is gonna be removed by Sentry automatically because of data scrubbing regexes (...token).

Reply:
🤞